jsign-crypto/src/test/java/net/jsign/KeyStoreBuilderTest.java

```diff
@@ -305,6 +305,38 @@ public void testBuildGaraSign() throws Exception {
         assertNotNull("keystore", keystore);
     }
 
+    @Test
+    public void testBuildVenafi() throws Exception {
+        KeyStoreBuilder builder = new KeyStoreBuilder().storetype(VENAFI);
+
+        Exception e = assertThrows(IllegalArgumentException.class, builder::build);
+        assertEquals("message", "storepass parameter must specify the Venafi username/password and/or the path to the keystore containing the TLS client certificate: <username>|<password>, <certificate>, or <username>|<password>|<certificate>", e.getMessage());
```

Comment:
Does this test really pass? The `validate()` method doesn't handle the third parameter with the client certificate.